scripts/integtest.sh

```diff
@@ -1,20 +1,18 @@
 #!/bin/bash
-
 set -e
-
 function usage() {
     echo ""
     echo "This script is used to run integration tests for plugin installed on a remote OpenSearch/Dashboards cluster."
     echo "--------------------------------------------------------------------------"
     echo "Usage: $0 [args]"
     echo ""
     echo "Required arguments:"
-    echo "None"
-    echo ""
-    echo "Optional arguments:"
-    echo -e "-b BIND_ADDRESS\t, defaults to localhost | 127.0.0.1, can be changed to any IP or domain name for the cluster location."
-    echo -e "-p BIND_PORT\t, defaults to 9200, can be changed to any port for the cluster location."
-    echo -e "-t TRANSPORT_PORT\t, defaults to 9300, can be changed to any port for the cluster location."
+    echo -e "-a Leader BIND_ADDRESS\t, defaults to localhost | 127.0.0.1, can be changed to any IP or domain name for the cluster location."
```

Comment:
Let's the keep the script argument names generic enough for other plugins as well. If any other plugin also needs more than one cluster, the logic to invoke their integtest.sh would be same.